### ISSUE TITLE
JDK missing or inaccessible folder in Ubuntu 18

### ISSUE THREAD

**Issue by flip-rahul** [redhat-developer/vscode-java#1044]
[provide a description of the issue]
The java.home variable defined in VS Code settings points to a missing or inaccessible folder (usr/lib/jvm/java-8-openjdk-amd64)
##### Environment
- Operating System: Ubuntu 18
- JDK version: java-8-openjdk-amd64
- Visual Studio Code version: 1.37.1
- Java extension version: 0.7.1

##### Steps To Reproduce
1. [step 1] Just open a java file and show this error. 
The java.home variable defined in VS Code settings points to a missing or inaccessible folder (usr/lib/jvm/java-8-openjdk-amd64)

##### Current Result 
showing Java file text in red color.

##### Expected Result 
I just want to open a java file in a normal java file way.

**Comment by Eskibear** [comment]
Looks you set the java.home by your own, did you miss the **leading slash `/`** of the path? `/usr/lib/jvm/java-8-openjdk-amd64` makes more sense to me.

**Comment by flip-rahul** [comment]
Yes i have entered the java.home by my own in settings.json file of VS code. I have tryed with leadind slash / but still issue is same.

**Comment by fbricon** [comment]
is javac present under /usr/lib/jvm/java-8-openjdk-amd64/bin?

**Comment by fbricon** [comment]
did you install VS Code using Flatpack?

**Comment by flip-rahul** [comment]
No there is no Javac in /usr/lib/jvm/java-8-openjdk-amd64/bin. I have Ubuntu 18. 
No i don't use Flatpack.

**Comment by fbricon** [comment]
So you're not using a JDK, but a JRE. Please install a JDK

**Comment by flip-rahul** [comment]
I just want to view a java file in VScode.

**Comment by fbricon** [comment]
vscode-java requires a JDK to work, so it can provide lots of Java smartness in VS Code. If you don't need it, just disable it, it'll stop complaining.

**Comment by flip-rahul** [comment]
is it possible to use java-8-openjdk-amd64 for this vascode-java. 
if possible then how?
How can i disable java complaining.

**Comment by akaroml** [comment]
> is it possible to use java-8-openjdk-amd64 for this vascode-java.
> if possible then how?
> How can i disable java complaining.

Yes, any JDK of version 8+ is ok for vscode-java to run. Just make sure you have it installed and your JAVA_HOME is pointing to it.

To help you set up the whole thing, you can run the command "Java: Configure Java Runtime" in VS Code to show the wizard.
